runner.sls

```diff
@@ -80,3 +80,14 @@ gitlab-runner:
         TimeoutStopSec={{ salt['pillar.get']('gitlab-runner:systemd-timeout-stop-sec') }}
         KillSignal=SIGQUIT
 {% endif %}
+
+{% if salt['pillar.get']('gitlab-runner:cloud-init', none) is not none %}
+# Deploy cloud-init file for autoscaling instances.
+/etc/gitlab-runner/autoscaler-cloudinit.yaml:
```

Comment:
Is this file in the directory automatically picket up by the autoscaler, or do we need to configure this somewhere, too? Quick Google didn't show up this file

Reply:
No you need to configure the in the `MachineOptions`. I added two examples to the file.